pkg/parsers/gitlab/common/runner_test.go

```diff
@@ -0,0 +1,58 @@
+package common
+
+import (
+	"testing"
+
+	gitlabModels "github.com/argonsecurity/pipeline-parser/pkg/loaders/gitlab/models/common"
+	"github.com/argonsecurity/pipeline-parser/pkg/models"
+	"github.com/argonsecurity/pipeline-parser/pkg/testutils"
+	"github.com/argonsecurity/pipeline-parser/pkg/utils"
+)
+
+func TestParseRunner(t *testing.T) {
```

Comment:
Images have multiple options:
* `registry/repo/name:tag`
* `repo/name:tag`
* `name:tag`
Test all of them